### PR TITLE
Prevent showing our "gpg-pubkey" pkg as unsigned

### PR DESCRIPTION
**Pull request by Rezney**
Our package "gpg-pubkey" is not signed and we are currently suggesting our users in the RedHatSignedRpmCheck actor to remove it even as it is our pkg.

This will also be needed if we would like to inhibit the upgrade with unsigned packages.

### CODE REVIEW

**Comment by vinzenz** [comment]
@Rezney you forgot to update the tests

File: repos/system_upgrade/el7toel8/actors/rpmscanner/actor.py [hunk 1]
```diff
@@ -22,18 +22,20 @@ def process(self):
             '/bin/rpm',
             '-qa',
             '--queryformat',
-            r'%{NAME}|%{VERSION}|%{RELEASE}|%|EPOCH?{%{EPOCH}}:{}||%|ARCH?{%{ARCH}}:{}||%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{(none)}|}|\n'
+            r'%{NAME}|%{VERSION}|%{RELEASE}|%|EPOCH?{%{EPOCH}}:{(none)}||%|PACKAGER?{%{PACKAGER}}:{(none)}||%|'
+            'ARCH?{%{ARCH}}:{}||%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{(none)}|}|\n'
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/rpmscanner/actor.py]: For the sake of consistency, add the r prefix also to this string.

File: repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/actor.py [hunk 2]
```diff
@@ -30,7 +30,8 @@ def process(self):
 
         for rpm_pkgs in self.consume(InstalledRPM):
             for pkg in rpm_pkgs.items:
-                if any(key in pkg.pgpsig for key in RH_SIGS):
+                if any(key in pkg.pgpsig for key in RH_SIGS) or \
+                    (pkg.name == 'gpg-pubkey' and pkg.packager.startswith('Red Hat, Inc.')):
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/actor.py]: Indent by additional 4 spaces ([flake8 E128](https://lintlyci.github.io/Flake8Rules/rules/E128.html)).

Comment (bocekm) [reply repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/actor.py]: Also add a comment to this line as the reason may not by obvious to everybody.

**Comment by Rezney** [comment]
@bocekm Thanks for the review. Will take a look at it on Friday.

**Comment by Rezney** [comment]
aah, yep also tests...

**Comment by Rezney** [comment]
@bocekm should be ok now

**Pull request merged**